src/components/message/message-test.stories.tsx

```diff
@@ -55,3 +55,25 @@ Default.args = {
   children: "This is some information from the Message Component.",
   showCloseIcon: true,
 };
+
+export const Transparent = (args: MessageProps) => {
+  const [isOpen, setIsOpen] = useState(true);
+  return (
+    <>
```

Comment:
**comment** (non-blocking): you don't need the fragment here if you're just rendering one component, you can just render the `Message` component